src/components/login/login.ts

```diff
@@ -1,49 +1,87 @@
-import { Component, EventEmitter, Output } from '@angular/core';
+import { Component, EventEmitter, Output } from "@angular/core";
 import { NavController } from "ionic-angular";
 import { AuthService } from "../../services/auth.service";
 import { UserDashboardPage } from "../../pages/user-dashboard/user-dashboard";
-import { GoogleAnalytics } from '../../application/helpers/GoogleAnalytics';
-
+import { GoogleAnalytics } from "../../application/helpers/GoogleAnalytics";
 
 @Component({
-  selector: 'login',
-  templateUrl: 'login.html'
+  selector: "login",
+  templateUrl: "login.html",
 })
-
 export class LoginComponent {
-
-	user: any = {};
-	private errors: any = [];
+  user: any = {};
+  counter: any = 0;
```

Comment:
counter seems to be only numbers from what I can see, is there a reason to use any specifically?

Reply:
Done